src/index.html

```diff
@@ -4,14 +4,149 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="container-field">
+        <legend>Personal information</legend>
+        <label for="surname" class="form-field">Surname:
+          <input type="text"
+                 name="surname"
+                 autocomplete="off"
+                 required
+                 class="input"
+          >
```

Comment:
Check your code using the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix this issues everywhere:
```suggestion
        <label for="surname" class="form-field">
          Surname:
          <input 
            type="text"
            name="surname"
            autocomplete="off"
            required
            class="input"
          >
```